### DISCUSSION TITLE
flinkcdc抽取ob for oracle的nvarchar字段乱码

### DISCUSSION THREAD

**Discussion by wanyujichun** [oceanbase/oceanbase#1374] [Q&A]
<img width="1402" alt="截屏2023-05-06 15 19 19" src="https://user-images.githubusercontent.com/8925285/236610101-0eb7aad6-d29a-40a0-8f67-3b85523041c5.png">

有大佬帮忙指导下如何解决吗？

**Comment by hnwyllmm** [comment]
refer https://github.com/oceanbase/flink-connector-oceanbase/discussions/22
